zigzag/data/molecule_junit.xsd

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <!--SIMPLE TYPES-->
+
+    <xs:simpleType name="validTestcaseProperty">
+        <xs:restriction base="xs:normalizedString">
+            <xs:enumeration value="test_id" />
+            <xs:enumeration value="jira" />
+            <xs:enumeration value="start_time" />
+            <xs:enumeration value="end_time" />
+        </xs:restriction>
+    </xs:simpleType>
+    <!--SIMPLE TYPES-->
```

Comment:
I think you grabbed the wrong file. On [master of pytest-zigzag](https://github.com/rcbops/pytest-zigzag/blob/master/pytest_zigzag/data/molecule_junit.xsd#L6) the `test_step` property is defined which is not here.